README.rst

```diff
@@ -448,3 +448,33 @@ and execute the following command:
 .. code-block:: console
     ~ # ping -c 3 192.168.0.2
 
+RPI5 vchan simple echo server
+*****************************
+
+vchan echo server sould be build with Kconfig configuration:
```

Comment:
sould -> should. Please enable spell checker and check spelling. Stopping review for now until you fix spelling

Reply:
Fixed